examples/freertos/CMakeLists.txt

```
@@ -54,14 +54,30 @@ set(FREERTOS_NAMESPACE FreeRTOS)
 if(USE_CMSIS_RTOS)
     list(APPEND CMSIS_COMP_LIST RTOS)
 endif()
+
 if(USE_CMSIS_RTOS_V2)
     list(APPEND CMSIS_COMP_LIST RTOS_V2)
 endif()
 
+if(FREERTOS_F100RB_EXAMPLE)
+    list(APPEND CMSIS_COMP_LIST STM32F1)
+    list(APPEND HAL_COMP_LIST STM32F1)
```

To be aligned with other example shouldn't it be like this ? To be honest I'm not sure it changes anything
```suggestion
    list(APPEND CMSIS_COMP_LIST STM32F100RB)
    list(APPEND HAL_COMP_LIST STM32F100RB)
```